docs/docs/auto-adjusting-aws-budget.md

```diff
@@ -0,0 +1,14 @@
+---
+title: Auto-adjusting AWS Budget Alerts
+description: "Overview of the Auto-adjusting Budget Alerts"
+layout: playbook_js
+tags: [playbook] 
+---
+
+![AWS Budget Alert](./images/auto-adjusting-aws-budget.png)
+
+AWS Budgets are a part of AWS Billing. Budgets are configured to send alert emails when we exceed specified amounts or when we’re forecast to exceed them. Currently we have two active budgets for all expenditures, one for actual expenditure and the other for forecast expenditure.
```

Comment:
Instead of "specified amounts", should add something about the customisable thresholds and that it can be set on the module for both budget and forecast